main.nf

```diff
@@ -308,7 +308,27 @@ if (params.expdesign)
 {
     Channel
         .fromPath(params.expdesign)
-        .set { ch_expdesign }
+        .set { ch_expdesign_pre }
+
+    process expdesign_raw2mzml {
+
+    label 'process_very_low'
+    label 'process_single_thread'
+
+    echo true
```

Comment:
Is it really required here?

Or was this just for you to debug?

Reply:
Ah thanks yes debug

Reply:
NP. Feel free to merge afterwards. LGTM.